integrations/client/test_delphi_epidata.py

```diff
@@ -38,69 +39,41 @@ def setUp(self):
     # use the local instance of the Epidata API
     Epidata.BASE_URL = 'http://delphi_web_epidata/epidata/api.php'
 
+    # use the local instance of the epidata database
+    secrets.db.host = 'delphi_database_epidata'
+    secrets.db.epi = ('user', 'pass')
+
   def tearDown(self):
     """Perform per-test teardown."""
     self.cur.close()
     self.cnx.close()
 
   def test_covidcast(self):
     """Test that the covidcast endpoint returns expected data."""
-    self.maxDiff=None
 
     # insert dummy data
     self.cur.execute('''
       insert into covidcast values
         (0, 'src', 'sig', 'day', 'county', 20200414, '01234',
           123, 1.5, 2.5, 3.5, 456, 4, 20200414, 0, 0, False),
         (0, 'src', 'sig2', 'day', 'county', 20200414, '01234',
-          123, 1.5, 2.5, 3.5, 456, 4, 20200414, 0, 0, False),
+          123, 1.5, 2.5, 3.5, 456, 4, 20200414, 0, 1, False),
         (0, 'src', 'sig', 'day', 'county', 20200414, '01234',
           456, 5.5, 1.2, 10.5, 789, 0, 20200415, 1, 0, False),
         (0, 'src', 'sig', 'day', 'county', 20200414, '01234',
           345, 6.5, 2.2, 11.5, 678, 0, 20200416, 2, 1, False)
     ''')
     self.cnx.commit()
 
-    # fetch data
```

Comment:
looks like git diff is having a really hard time with the lines below. there's no functional change, it's just:

- wrap each request/assert in a `subTest` for better output/debugging
- indent each by 2 (since `subTest` is a `with` block)
- wrap one row that was too long